src/main/java/frc/robot/subsystems/TiltedElevatorSubsystem.java

```diff
@@ -167,6 +168,11 @@ public void periodic() {
             return;
         }
 
+        if (flipCone){
```

Comment:
i think instead of having this boolean we could add an elevator state like `SLOWLY_LOWER` and run custom logic if that's our target?